### PR TITLE
KUBE-566: refactor before multi cmd

### PR DESCRIPTION
**Pull request by ValyaB**

### CODE REVIEW

File: .github/workflows/golangci-lint.yaml [hunk 1]
```diff
@@ -10,9 +10,9 @@ jobs:
     steps:
       - uses: actions/setup-go@v5
         with:
-          go-version: '1.22.7'
-      - uses: actions/checkout@v3
+          go-version: 'stable'
```
Comment (Trojan295) [inline .github/workflows/golangci-lint.yaml:13]: Is there some reason to use `stable` instead of a fixed version?

File: main.go [hunk 2]
```diff
@@ -28,14 +28,15 @@ import (
...
-	"github.com/castai/cluster-controller/castai"
-	"github.com/castai/cluster-controller/config"
-	"github.com/castai/cluster-controller/csr"
 	"github.com/castai/cluster-controller/health"
-	"github.com/castai/cluster-controller/helm"
-	"github.com/castai/cluster-controller/version"
-	"github.com/castai/cluster-controller/waitext"
+	"github.com/castai/cluster-controller/internal/actions/csr"
+	"github.com/castai/cluster-controller/internal/castai"
+	config2 "github.com/castai/cluster-controller/internal/config"
```
Comment (Trojan295) [inline main.go:34]: Why `config2`?

**Review by Trojan295 — APPROVED** [review]

**Review by stgleb — APPROVED** [review]
LGTM

File: main.go (hunk 2 above)
Comment (ValyaB) [reply main.go:34]: good catch, let me fix it (IDE did refactor on moving packages)

**Pull request merged**